src/main/java/com/revature/studyforce/stacktrace/dto/SolutionDTO.java

```diff
@@ -22,12 +18,11 @@
 @AllArgsConstructor
 public class SolutionDTO {
     private int solutionId;
-    private Stacktrace stackTraceId;
-    private User userId;
+    private int stackTraceId;
+    private int userId;
```

Comment:
This should be a custom StacktraceUserDTO or something. So that the JSON contains the username.

Reply:
This is so we can desplay the submitting user on the solution without another call to the database.